samples/test/quickstart.js

```diff
@@ -21,30 +21,25 @@ const path = require('path');
 const cp = require('child_process');
 const {before, describe, it} = require('mocha');
 // eslint-disable-next-line node/no-missing-require
-const {ServiceManagerClient} = require('');
+const {ServiceManagerClient} = require('@google-cloud/service-management');
 // eslint-disable-next-line no-unused-vars, node/no-missing-require
 const {assert} = require('chai');
 
 const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
 
 const cwd = path.join(__dirname, '..');
 
-const client = new {ServiceManagerClient}();
+const client = new ServiceManagerClient();
 
 describe('Quickstart', () => {
-  //TODO: remove this if not using the projectId
-  // eslint-disable-next-line no-unused-vars
   let projectId;
 
   before(async () => {
-    // eslint-disable-next-line no-unused-vars
     projectId = await client.getProjectId();
   });
 
   it('should run quickstart', async () => {
-    //TODO: remove this line
-    // eslint-disable-next-line no-unused-vars
-    const stdout = execSync('node ./quickstart.js', {cwd});
-    //assert(stdout, stdout !== null);
+    const stdout = execSync(`node ./quickstart.js ${projectId}`, {cwd});
+    assert.match(stdout, /\[[\s\S]*\]/);
```

Comment:
we're testing that an empty array of services were returned? perhaps ignore my prior comment about printing each individual one 👍

Reply:
Yep, exactly!